.github/release.yml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  packages: write
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: "remear/reactor-developer-docs"
+
+jobs:
+  docker:
```

Comment:
Same thoughts as above. What do you think about naming this `build-release` for now?